backend/migrations/versions/2c4d0d16f601_create_test_cases_table.py

```
@@ -0,0 +1,44 @@
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('input', sa.Text(), nullable=False),
+    sa.Column('output', sa.Text(), nullable=False),
+    sa.Column('module_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['module_id'], ['modules.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('user_test_cases',
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('test_case_id', sa.Integer(), nullable=False),
+    sa.Column('verified', sa.Boolean(), nullable=True),
+    sa.ForeignKeyConstraint(['test_case_id'], ['test_cases.id'], ),
```

Ruff (0.8.2)

Trailing comma prohibited

Remove trailing comma

(COM819)